guilford_court_commitment.yaml

```diff
@@ -276,9 +234,8 @@ associationDefinitions:
       ol.id:
         type: "ol.id"
         transforms:
-        - !<transforms.HashTransform>
-          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "OFFENDER ADMISSION/INTAKE DATE"]
-          hashFunction: "sha256"
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "OFFENDER ADMISSION/INTAKE DATE"]
```

Comment:
In this earlier flight, we added a "1" and hashed it, it seems like we should do the same here so that identical data merges. Even though we decided not to hash things as much anymore. https://github.com/Lattice-Works/Guilford-Co-NC/blob/7b7f738fff953547e6d1bd582d41334c05aef4fc/guilford_inmates_dps.yaml#L1021